staging/src/k8s.io/api/OWNERS

```diff
@@ -13,3 +13,7 @@ filters:
   "\\.go$":
     labels:
       - kind/api-change
+  # allow dependency approver(s) for just go.mod/go.sum files.
```

Comment:
note that prow doesn't actually have per-file approvers implemented AFAIK

Reply:
So is this all moot?

Reply:
Damn, i keep forgetting @BenTheElder!